readme.md

```diff
@@ -9,3 +9,16 @@ The SQLRace API is used to read and write time-series data stored in supported d
 SQLRace API is available as a Nuget package to registered users from the **[McLaren Applied Nuget Repository](https://github.com/mat-docs/packages)**. 
 
 See the [API Documentation](https://mat-docs.github.io/)
+
+## Notes on upgrading MESL.SQLRace.API package from versions prior to 2.1.22127.1
+
+Version 2.1.22127.1 of MESL.SQLRace.API upgrades System.Reactive from 3.1.1 to 4.4.1. If you are developing your solution in Visual Studio then be advised that the Nuget upgrade process does not automatically remove any previously dependent packages that are no longer required. 
+In order to reduce the likelihood of runtime errors we strongly recommend you manually remove (in the order specified) these redundant packages following the upgrade:
+
+1. System.Reactive.Interfaces 3.1.1
```

Comment:
Really, I thought the others depended on this and so this should be removed last?